$OpenBSD: patch-lib_XML_TreePP_pm,v 1.1.1.1 2012/01/24 00:25:56 nigel Exp $

timeout of 10 seconds is way too low, stick with defaults

--- lib/XML/TreePP.pm.orig	Sun Oct 31 10:47:03 2010
+++ lib/XML/TreePP.pm	Sun Jan 16 14:40:59 2011
@@ -627,7 +627,6 @@ sub parsehttp_lwp {
     my $ua = $self->{lwp_useragent} if exists $self->{lwp_useragent};
     if ( ! ref $ua ) {
         $ua = LWP::UserAgent->new();
-        $ua->timeout(10);
         $ua->env_proxy();
         $ua->agent( $self->{__user_agent} ) if defined $self->{__user_agent};
     } else {
